services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -264,19 +268,19 @@ async def get_file(
 
 @router.delete(
     "/{file_id}",
```

Comment:
is the 204 automatically set in the documentation now?

Reply:
I removed the status code. I think it was originally put there because the endpoint hadn't been implemented yet.